docu/files/Multi-tabEditor.html.md

```diff
@@ -0,0 +1,96 @@
+---
+title: 'The multi-tab editor component in the Builder'
+service: 'Builder SDK'
+area: 'Core'
+type: Tutorial
+order: 4
+---
+
+The multi-tab editor is a custom Angular directive in **builder_editors** with a dependency to the **builder**.
```

Comment:
Per the style guide, "the Builder" always carries a lowercase "the" and capitalized "Builder," without additional formatting, as shown: 
The multi-tab editor is a custom Angular directive in **builder_editors** with a dependency on the Builder.